fairroot.sh

```diff
@@ -70,6 +70,13 @@ cmake $SOURCEDIR
 
 cmake --build . -- -j$JOBS install
 
+INCLUDE_TREE="$INSTALLROOT/include_tree"
```

Comment:
Can we do this inside `include` without creating a new directory?

Reply:
We can, but then we pollute the "real" include directory (which is picked up by the build systems of other packages) with spurious header paths.

Reply:
In principle if something (like CMake) looks for a header under `include` it will find it many times now, but the first one should win. Let's see if the test build works.